packages/clay-css/src/scss/components/_list-group.scss

```diff
@@ -409,21 +409,26 @@
 
 // List Group Item Variants
 
-@each $color, $value in $theme-colors {
+@each $color, $value in $list-group-item-theme-colors {
```

Comment:
@pat270 Couldn't this be a breaking change or was `$theme-colors` only used internally?

Reply:
@matuzalemsteles Yes this is a breaking change since changing `$theme-colors` no longer affects this component. The way `$theme-colors` generates variant colors was so bad I doubt anyone was reliably using this.

Reply:
Hmm I see, it makes sense not to link, well I'll trust you 🙂, anyway in case anyone missed it, we can recommend replacing it because `theme-colors` has problems accepting CSS variables as arguments.